public/js/model/index.js

```diff
@@ -28,6 +28,23 @@
 
     });
 
+    $('form.filters').on('submit', function (event) {
```

Comment:
Nothing wrong with this line but I think you are missing a version increment for `linz.css` somewhere.

Reply:
Done.